example_project/run_simple_workflow.py

```diff
@@ -0,0 +1,235 @@
+from pprint import pprint
+from time import sleep
+
+import argparse
+import os
+import subprocess
+import sys
+
+# Set up the standalone Django script
+proj_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                         "../")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "example_project.settings")
+sys.path.append(proj_path)
+from django.core.wsgi import get_wsgi_application
+application = get_wsgi_application()
+
+from orchestra.orchestra_api import create_orchestra_project
+from orchestra.orchestra_api import get_project_information
+
+FAST_MODE = False
+
+
+def pause(n_seconds):
+    if not FAST_MODE:
+        sleep(n_seconds)
+
+
+def run_demo():
+    intro()
+    describe_workflow()
+    project_id = create_project()
+    project_info_1(project_id)
+    rating_task()
+    project_info_2(project_id)
+    conclusion()
+
+
+def intro():
+    subprocess.call('clear')
+    print('----------------------------------------------')
+    print('Welcome to the Orchestra Simple Workflow Demo.')
+    print('----------------------------------------------')
+    pause(2)
+    print('In this demo, we will create a new project using the "Simple '
+          'Workflow" workflow,')
+    pause(2)
+    print('which automatically extracts an image from a '
+          'url and asks a human expert to rate it.')
+    pause(2)
+    ack = input('Are you ready to begin? (y/n): ').lower()
+    while ack not in ['y', 'n']:
+        ack = input('Please respond with "y" or "n": ').lower()
+    if ack == 'n':
+        print('Ok, maybe some other time!')
+        sys.exit()
+    print('Great!')
+
+
+def describe_workflow():
+    print('')
+    print('The Simple Workflow has two steps, one automated and one human.')
+    pause(2)
+    print('The automated step takes a URL and extracts a random image from '
+          'the page.')
+    pause(2)
+    print('The human step asks an expert to rate how "awesome" the image is '
+          'on a scale from one to five.')
+    pause(2)
+    print('For this demo, we will scrape an image from www.josephbotros.com, '
+          'the homepage of one of our rockstar developers.')
```

Comment:
Went with `visionary` per popular agreement, though it turns out @jrbotros really liked `rockstar`.